docgenhtml/plugins/org.polarsys.capella.docgen/src/org/polarsys/capella/docgen/util/CapellaLabelProviderHelper.java

```diff
@@ -0,0 +1,26 @@
+/*******************************************************************************
+ * Copyright (c) 2020 Obeo.
```

Comment:
The copyright is wrong. Please replace it with the Thalesgroup copyright

Reply:
Done